public/css/index.css

```diff
@@ -31,23 +31,18 @@ pre.wrap,
 code.wrap {
```

Comment:
@tboerger was my fault, If you see on PR https://github.com/gogits/gogs/pull/3786, don't have empty lines on css, I don't know why, my npm display that less version is 2.7.1, but on my terminal was using 1.4.2. Now I solved this and generated correct css files, with no empty files and with `!important` as defined on less.